src/leader.py

```diff
@@ -63,13 +63,21 @@ def turn_straight(self):
         self.fw.turn(self.STRAIGHT_ANGLE)
 
 
-    def turn(self,position):
-        if position == -1:
-            self.turn_left()
-        elif position == 1:
-            self.turn_right()
-        else:
-            self.turn_straight()
+    def set_turn_angle(self,position):
+        self.fw.turn(89 + (45  * position))
```

Comment:
What is 89? What is 45? Wisam asked you to remove the magic numbers.

Reply:
89 is the angle where the car faces forwards and the 45 degrees is the largest turn the car can make, so I multiplied that value by the position which is a decimal from -1 to 1 in order to get teh angle we turn at. Also what do you mean by magic number?